.github/workflows/release-ts-api-package.yml

```diff
@@ -2,31 +2,37 @@
 name: Release Ts API Package
 
 on:
+  push:
+    branches:
+      - dev
   workflow_dispatch:
       inputs:
         tag:
+          description: 'Package release tag'
+          required: true
+          default: 'dev'
+        release:
           description: 'Release docker image tag'
           required: true
           default: 'latest'
 env:
   NODE_AUTH_TOKEN: ${{ secrets.RELEASE_TS_API_PACKAGE_TOKEN }}
 
 jobs:
-        
   update-publish:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
       - name: Set env variables
-        run: echo "RELEASE_IMAGE_VERSION=${{ inputs.tag }}" >> "$GITHUB_ENV"
+        run: echo "RELEASE_IMAGE_VERSION=${{ inputs.release }}" >> "$GITHUB_ENV"
```

Comment:
Here `inputs.release` used. Line 83 `inputs.tag` used. Can you explain a bit?

Reply:
The separation of the inputs helps to pick `parachain/worker` with `release` input, build the stuff and release it with specific tag. For example with latest `parachain/worker` we can publish `dev/alpha` release of `client-api`. Before it was possible to publish only the same versions.